<?xml version="1.0" encoding="UTF-8"?>
<testsuite hostname="Elan-thinkpad" name="com.mycompany.testng.StringUtilsTest" tests="1" failures="1" timestamp="29 Aug 2013 06:54:23 GMT" time="0.026" errors="0">
  <testcase name="print" time="0.01" classname="com.mycompany.testng.StringUtilsTest">
    <failure type="org.testng.TestNGException" message="
Can&amp;apos;t invoke public void com.mycompany.testng.StringUtilsTest.print(java.lang.String): either make it static or add a no-args constructor to your class">
      <![CDATA[org.testng.TestNGException: 
Can't invoke public void com.mycompany.testng.StringUtilsTest.print(java.lang.String): either make it static or add a no-args constructor to your class
	at org.testng.internal.Utils.checkInstanceOrStatic(Utils.java:789)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:36)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:676)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1169)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1182)
	at org.testng.TestRunner.privateRun(TestRunner.java:761)
	at org.testng.TestRunner.run(TestRunner.java:612)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:292)
	at org.testng.SuiteRunner.run(SuiteRunner.java:241)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1169)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1094)
	at org.testng.TestNG.run(TestNG.java:1006)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:107)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:199)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:170)
]]>
    </failure>
  </testcase>
</testsuite>
